### ISSUE TITLE
Error while creating multiple responses

### ISSUE THREAD

**Issue by taromero** [spreaker/grunt-aws-apigateway#1]
I'm getting this error most of the times (a few times it has succeeded) when having at least 4 responses. For example:

``` javascript
responses: {
  200: {
    // Pass-through
    responseModels: {
      'application/json': 'Empty'
    }
  },
  400: {
    selectionPattern: '.*status_400.*'
  },
  404: {
    selectionPattern: '.*status_400.*'
  },
  500: {
    selectionPattern: '.*status_500.*'
  }
}
```

The error message is:

`Warning: Unable to create method response GET for resource /assets/publishers/{publisher_id}: Unable to complete operation due to concurrent modification. Please try again later. Use --force to continue.`

**Comment by pracucci** [comment]
Published version `0.0.4` that includes your fix. Thanks!

**Comment by taromero** [comment]
Thanks for the quick response, and this great tool!
